### ISSUE TITLE
NullPointerException in KFreonLib.dll trying to find a game directory that does not exist.

### ISSUE THREAD

**Issue by Mgamerz** [ME3Explorer/ME3Explorer#7]
An unhandled exception of type 'System.NullReferenceException' occurred in KFreonLib.dll

Additional information: Object reference not set to an instance of an object.

```
        get // if you are trying to use gamePath variable and it's null it asks to locate ME3 exe file
        {
            /*if (_gamePath == null)
                _gamePath = KFreonLib.Misc.Methods.SelectGameLoc(3);
            else */if (!_gamePath.EndsWith("\\"))
                _gamePath += '\\';
            return _gamePath;
        }
```

Gamepath is null. Tested by loading on a machine that does not have ME3 installed.

**Comment by Mgamerz** [comment]
In ME3Directory.cs.

**Comment by HeffU** [comment]
K has fixed this in his branch, I added the same fix in the cleanup-and-fixes branch.
I'll close this once that's merged into master.

**Comment by HeffU** [comment]
Should be fixed in 56e0a0cb96b24ad030c6aa241b01d1543c87e4e8
